koku/api/report/constants.py

```diff
@@ -10,3 +10,8 @@
 AND_AWS_CATEGORY_PREFIX = "and:aws_category:"
 OR_AWS_CATEGORY_PREFIX = "or:aws_category:"
 URL_ENCODED_SAFE = "[]:"
+AWS_MARKUP_COST = {
```

Comment:
Is it intentional, that we don't have "blended_cost" here?

Reply:
This is incorrect. Thank for catching this!